### ISSUE TITLE
Cannot compile file serenity

### ISSUE THREAD

**Issue by sinchr** [VirtualBrightPlayz/DiscordStatusCB#1]
Hello, I cannot compile file serenity. What should I do?
![Screenshot_5](https://user-images.githubusercontent.com/119080264/204055866-f05b6279-47c4-46bb-bcce-44429959239a.png)

**Comment by VirtualBrightPlayz** [comment]
Do not use `cargo run`, use `cargo build --target=i686-pc-windows-msvc --release` and copy the dll next to the game directory.

**Comment by sinchr** [comment]
Still something
![Screenshot_6](https://user-images.githubusercontent.com/119080264/204060522-79f1a00d-5c52-4408-9c93-c76baecf9302.png)

**Comment by VirtualBrightPlayz** [comment]
Run `rustup target add i686-pc-windows-msvc` and then `cargo build --target=i686-pc-windows-msvc --release`

**Comment by sinchr** [comment]
![Screenshot_7](https://user-images.githubusercontent.com/119080264/204090852-f86dc7f2-c61c-4148-a3e8-237b82e59a3f.png)

**Comment by sinchr** [comment]
![Screenshot_8](https://user-images.githubusercontent.com/119080264/204093223-c0254f36-de91-432d-8ce8-96a49fc6f6aa.png)

**Comment by VirtualBrightPlayz** [comment]
You need to install rustup first before you run it. https://www.rust-lang.org/tools/install

**Comment by sinchr** [comment]
Idk. what should i do?
![Screenshot_9](https://user-images.githubusercontent.com/119080264/204343620-d28d0c83-0fc6-41d4-87a2-c94276acf067.png)

**Comment by VirtualBrightPlayz** [comment]
You must run these on a windows pc or msvc-like environment

**Comment by sinchr** [comment]
Okay, I think I get it. What file should I save it as?
![Screenshot_10](https://user-images.githubusercontent.com/119080264/205109077-d6393398-b023-4b67-bc97-a118031c455b.png)

**Comment by VirtualBrightPlayz** [comment]
`discord_bot.dll` will be in the folder `rustbot\target\i686-pc-windows-msvc\release` after a successful compile. Hope this helps!
EDIT1: The dll goes next to the server exe

**Comment by sinchr** [comment]
It's there
![Screenshot_1](https://user-images.githubusercontent.com/119080264/205349529-2f9ee5e0-5a38-4170-9e30-a62dba6fcc12.png)
Now it's saying something
![Screenshot_2](https://user-images.githubusercontent.com/119080264/205353369-a7b9ec51-1679-4086-9e44-ed8fb1424b2b.png)
But i cannot join server (Server not responding)

**Comment by VirtualBrightPlayz** [comment]
Seems like something with the hosting provider, zap, because it should work fine. Is there a discord folder with the bot token and all that?

**Comment by sinchr** [comment]
Is that right?
![Screenshot_33](https://user-images.githubusercontent.com/119080264/205453562-b9efa1ba-ba0c-4063-b200-c1c461ffb6c4.png)

**Comment by VirtualBrightPlayz** [comment]
Yes, sorry for the late response. It seems like your hosting provider (zap!) does not allow for the dll to be loaded.
